app/models/verification.rb

```diff
@@ -13,6 +13,6 @@ class Verification < ApplicationRecord
   private
 
   def send_to_suggestions_store
-    UpdateStatementVerification.run(self) if statement.from_suggestions_store?
+    UpdateStatementVerification.run(self) if statement.page.from_suggestions_store?
```

Comment:
Does this (and the change below in `StatementClassifier`) need to ne changed? I think if you're delegating in the `Statement` model then you can leave this as it was?

Reply:
Oh yes, good point - in an early version I just moved it and added the delegation later, then forgot that they could stay the same.